tests/ghc.nix

```diff
@@ -6,6 +6,7 @@ let haskellPackages = pkgs.haskell.packages.ghc822.override {
       overrides = with pkgs.haskell.lib; self: super: rec {
         lens-labels = super.lens-labels_0_2_0_1;
         proto-lens = super.proto-lens_0_3_1_0;
+        libc = import ./haddock/libC.nix self pkgs;
```

Comment:
This needs to be solved differently; my suggestion is creating a different `ghcWithPackages` for the tests.